app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.appcompat.widget.LinearLayoutCompat xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/bottom_sheet_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:padding="10dp">
+
+    <TextView
+      android:id="@+id/action_options"
+      android:layout_width="match_parent"
+      android:layout_height="48dp"
+      android:drawablePadding="10dp"
+      android:gravity="center_vertical"
+      android:text="@string/menu_options"
+      android:textColor="@color/color_def_black"
```

Comment:
File an issue for `Dark Mode Support for bottom sheet` in both the places and mention it in your PR

Reply:
Added TODO for that and filed issue #4513